dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -172,6 +180,31 @@ public static ZenodoDoiResult registerZenodoDOI(ApiClient zenodoClient, Workflow
         return new ZenodoDoiResult(doiAlias, publishedDeposit.getMetadata().getDoi(), conceptDoi);
     }
 
+    public static List<String> findDOIsForGitHubRepo(ApiClient zenodoClient, String gitHubRepo) {
+        final ArrayList<String> dois = new ArrayList<>();
+        final PreviewApi previewApi = new PreviewApi(zenodoClient);
+        final String query = URLEncoder.encode('"' + gitHubRepo + '"');
+        final int pageSize = 100;
+        final SearchResult records = previewApi.listRecords(query, "bestmatch", 1, pageSize);
```

Comment:
Any significance to 100? 
Seems like we could get away with lower, especially if we ignore multiple DOIs for a repo as a first pass
(or is the search likely to get stuff from other repos?)

Reply:
It was a guess to start with. Because it's elastic search and not an exact match, which I can't figure out how to do, I figure cast a wider net and narrow down on a specific field. It will probably require some tinkering.